### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by DevN1mr0d**

### CODE REVIEW

File: locales/ro.yml [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+    footnote: "<b>Pro Tip:</b> Remember your password."
+    label_enter: New Password
+    label_re_enter: Re-Enter New Password
+    status_flash_success: Your password has been changed successfully.
+    subtitle: Not too short, not too long, just the…
+    title: Reset Password
+  select_language:
+    button: Change Default Language
+    help_translate: Visit our <a class="silver" href="https://github.com/fetlife/translations">translation project</a> on GitHub and help make FetLife accessible to more of our kinky friends <span class="amp">&amp</span> communities.
+    language_name: English
```
Comment (rakvat) [inline locales/ro.yml]: At least this line needs to be changed to the Language name.

**Comment by rakvat** [comment]
@DevN1mr0d thank you for your contribution. However, I feel like translating some more strings before adding  a new language would be good. Also, the tests are currently failing on this branch.

File: locales/ro.yml (hunk 1 above)
Comment (DevN1mr0d) [reply locales/ro.yml]: Thanks, sent a new pull request with some additional translation.

**Comment by DevN1mr0d** [comment]
I am clueless at the moment on this one. Line 696 where the error points is not even translated, the same as the original English file.

**Comment by nielsenramon** [comment]
Closing as it's missing our new strings. 

Recommendation: Smaller PRs so it's manageable to review from our end.

**Pull request closed without merging**